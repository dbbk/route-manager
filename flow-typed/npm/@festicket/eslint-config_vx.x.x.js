// flow-typed signature: 564086c0274f153b4c37ac5c954f219e
// flow-typed version: <<STUB>>/@festicket/eslint-config_v^1.1.5/flow_v0.63.1

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@festicket/eslint-config'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@festicket/eslint-config' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */


// Filename aliases
declare module '@festicket/eslint-config/index' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
declare module '@festicket/eslint-config/index.js' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
