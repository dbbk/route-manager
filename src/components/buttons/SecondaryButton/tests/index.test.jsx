import React from 'react';
import { mount } from 'enzyme';
import { SecondaryButton } from '@festicket/react-ui-components';

describe('<SecondaryButton />', () => {
  test('with initial props', () => {
    const tree = mount(<SecondaryButton />);
    expect(tree).toMatchSnapshot();
  });
});
