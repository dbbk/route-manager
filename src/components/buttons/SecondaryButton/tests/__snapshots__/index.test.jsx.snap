// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SecondaryButton /> Snapshot Bordered 1`] = `
.c1 {
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #8C64CB;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 16px;
  min-width: 200px;
  min-height: 50px;
  padding: 15px 30px;
}

.c0 {
  display: inline-block;
  -webkit-text-align: center;
  text-align: center;
  cursor: pointer;
  font-weight: 700;
  border-radius: 0;
  padding: 0;
  font-size: 0;
  color: #A2A4AD;
  background-color: rgba(36,37,41,0.05);
}

.c0:hover {
  background-color: rgba(36,37,41,0.1);
  color: #9497a1;
}

<a
  className="c0 c1"
  element="a"
  target="_blank"
  to="#"
  variant="standard"
  weight="bold"
>
  <div
    className="c2"
    size="regular"
  />
</a>
`;

exports[`<SecondaryButton /> Snapshot Bordered Small 1`] = `
.c1 {
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #8C64CB;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 16px;
  min-width: 75px;
  min-height: 40px;
  padding: 10px 15px;
}

.c0 {
  display: inline-block;
  -webkit-text-align: center;
  text-align: center;
  cursor: pointer;
  font-weight: 700;
  border-radius: 0;
  padding: 0;
  font-size: 0;
  color: #A2A4AD;
  background-color: rgba(36,37,41,0.05);
}

.c0:hover {
  background-color: rgba(36,37,41,0.1);
  color: #9497a1;
}

<a
  className="c0 c1"
  element="a"
  target="_blank"
  to="#"
  variant="standard"
  weight="bold"
>
  <div
    className="c2"
    size="small"
  />
</a>
`;

exports[`<SecondaryButton /> Snapshot Default 1`] = `
.c1 {
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #8C64CB;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 16px;
  min-width: 200px;
  min-height: 50px;
  padding: 15px 30px;
}

.c0 {
  display: inline-block;
  -webkit-text-align: center;
  text-align: center;
  cursor: pointer;
  font-weight: 700;
  border-radius: 0;
  padding: 0;
  font-size: 0;
  color: #A2A4AD;
  background-color: rgba(36,37,41,0.05);
}

.c0:hover {
  background-color: rgba(36,37,41,0.1);
  color: #9497a1;
}

<a
  className="c0 c1"
  element="a"
  target="_blank"
  to="#"
  variant="standard"
  weight="bold"
>
  <div
    className="c2"
    size="regular"
  />
</a>
`;

exports[`<SecondaryButton /> Snapshot Default Button Element 1`] = `
.c1 {
  border: none;
  outline: none;
  margin: 0;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 16px;
  min-width: 200px;
  min-height: 50px;
  padding: 15px 30px;
}

.c0 {
  display: inline-block;
  -webkit-text-align: center;
  text-align: center;
  cursor: pointer;
  font-weight: 700;
  border-radius: 0;
  padding: 0;
  font-size: 0;
  color: #A2A4AD;
  background-color: rgba(36,37,41,0.05);
}

.c0:hover {
  background-color: rgba(36,37,41,0.1);
  color: #9497a1;
}

<button
  aria-disabled={false}
  className="c0 c1"
  disabled={false}
  size="regular"
  to="#"
>
  <div
    className="c2"
    size="regular"
  />
</button>
`;

exports[`<SecondaryButton /> Snapshot Default Small 1`] = `
.c1 {
  cursor: pointer;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #8C64CB;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  font-size: 16px;
  min-width: 75px;
  min-height: 40px;
  padding: 10px 15px;
}

.c0 {
  display: inline-block;
  -webkit-text-align: center;
  text-align: center;
  cursor: pointer;
  font-weight: 700;
  border-radius: 0;
  padding: 0;
  font-size: 0;
  color: #A2A4AD;
  background-color: rgba(36,37,41,0.05);
}

.c0:hover {
  background-color: rgba(36,37,41,0.1);
  color: #9497a1;
}

<a
  className="c0 c1"
  element="a"
  target="_blank"
  to="#"
  variant="standard"
  weight="bold"
>
  <div
    className="c2"
    size="small"
  />
</a>
`;
